src/Sql/Sql/help/Get-AzSqlDatabaseSensitivityClassification.md

```diff
@@ -0,0 +1,283 @@
+```
+
+### -DatabaseName
+The name of the Azure SQL database.
+
+```yaml
+Type: System.String
+Parameter Sets: DatabaseParameterSet, ColumnParameterSet
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DatabaseObject
+The SQL database object.
+
+```yaml
+Type: Microsoft.Azure.Commands.Sql.Database.Model.AzureSqlDatabaseModel
+Parameter Sets: DatabaseObjectParameterSet, DatabaseObjectColumnParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The name of the resource group.
+
+```yaml
+Type: System.String
+Parameter Sets: DatabaseParameterSet, ColumnParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SchemaName
+Name of schema.
+
+```yaml
+Type: System.String
+Parameter Sets: ColumnParameterSet, DatabaseObjectColumnParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ServerName
+SQL server name.
+
+```yaml
+Type: System.String
+Parameter Sets: DatabaseParameterSet, ColumnParameterSet
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -TableName
+Name of table.
+
+```yaml
+Type: System.String
+Parameter Sets: ColumnParameterSet, DatabaseObjectColumnParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Microsoft.Azure.Commands.Sql.Database.Model.AzureSqlDatabaseModel
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.Sql.DataClassification.Model.SqlDatabaseSensitivityClassificationModel
+
+## NOTES
+
+## RELATED LINKS
+
+Learn more about [Azure SQL Database data discovery and classification](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-data-discovery-and-classification)
```

Comment:
@bashahee the help generation module, `platyPS`, is expecting the lines provided in this section follow a specific format of strictly hyperlink text and nothing else. I believe the build failures are resolved if you more the "Learn more about" text into the brackets with the "Azure SQL Database data discovery and classification" text. You do can this, or you can move the entire line into the `## NOTES` section. This should be done for every markdown file where you add this "Learn more about" link. Please let us know if you have any questions.